docs/00_Prepare_everything.md

```diff
@@ -15,16 +15,16 @@ First and foremost, you need to meet the following conditions in order to write
 
 ### What You Need to Know:
```

Comment:
Why Do We Put Caps On All Titles?

Reply:
We do not put caps on all words of a  titles. The Chicago style was used here https://en.wikipedia.org/wiki/Capitalization_in_English#Title_capitalization_in_different_styles